support/cas-server-support-surrogate-authentication/src/test/java/org/apereo/cas/config/BaseSurrogateAuthenticationTestsConfiguration.java

```diff
@@ -0,0 +1,25 @@
+package org.apereo.cas.config;
+
+import org.apereo.cas.services.ServicesManager;
+
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.cloud.context.config.annotation.RefreshScope;
+import org.springframework.context.annotation.Bean;
+
+import static org.mockito.Mockito.*;
+
+/**
+ * This is {@link BaseSurrogateAuthenticationTestsConfiguration}.
+ *
+ * @author Timur Duehr
+ * @since 6.0.0
+ */
+@TestConfiguration("casSurrogateAuthenticationServiceTestConfiguration")
+public class BaseSurrogateAuthenticationTestsConfiguration {
```

Comment:
This will likely need to go away. IIRC, there is a test configuration class already and mocks up the service manager with a number of dummy/test service definitions. This may be an opportunity to reuse that. If not, then InjectMocks is probably an easier thing to do and it's less LOC.